features/step_definitions/custom_web_steps.rb

```diff
@@ -88,7 +88,7 @@ def take_screenshots_with_login
 end
 
 When /^I prepare the deletion of the first post$/ do
-  within(find('.stream .stream_element')) do
+  within(find('.stream .stream_element', match: :first)) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.